src/renderer/components/execute/TableDropdown.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import { Dropdown } from 'react-bootstrap';
+
+import '../../scss/Execute.scss';
+
+interface ITableDropdownProps {
+  activeDb: string;
+}
+
+const TableDropdown = ({ activeDb }: ITableDropdownProps) => {
+  const tablesDefault = ['test1', 'test2', 'test3'];
+  const [tables, setTables] = useState<string[]>();
+  const [activeTable, setActiveTable] = useState<string>('Select Table');
+  useEffect(() => {
+    const fetchTables = async () => {
+      const newTables = await window.procedures.ipcRenderer.fetchTables();
+      setTables(newTables);
+    };
+    fetchTables();
```

Comment:
```suggestion
    fetchTables();
    setActiveTable('');
```
This way the table string gets reset whenever the database changes